modules/world.py

```diff
@@ -63,6 +66,21 @@ def exit(self):
             return True
         else:
             return False
+			
+    def pick_up_item(self):
+        key_x = self.data["key"]['x']
+        key_y = self.data["key"]['y']
+        gold_x = self.data["gold"]['x']
+        gold_y = self.data["gold"]['y']
+        x, y = self.locate("player")		
+               
+        
```

Comment:
I was trying to use locate first and it kept returning an error saying its unable to be hashed. Not totally sure why that happened, but I forgot about Room.items(x, y). Also, you're right the purpose of this method was to check the player location against the item location, if True, add item to bag. 
Jonathan Colón 

> On Jun 28, 2015, at 08:22, Jamal Hansen <notifications@github.com> wrote:
> 
> In modules/world.py:
> 
> > @@ -63,6 +66,21 @@ def exit(self):
> >              return True
> >          else:
> >              return False
> > +			
> > +    def pick_up_item(self):
> > +        key_x = self.data["key"]['x']
> > +        key_y = self.data["key"]['y']
> > +        gold_x = self.data["gold"]['x']
> > +        gold_y = self.data["gold"]['y']
> > +        x, y = self.locate("player")		
> > +               
> > +        
> There is locate(), but what it really looks like you are trying to figure out is what is in the same tile as the player. Luckily that exists too :)
> 
> Room.items(x,y) will tell you everything at those co-ordinates including the player
> 
> —
> Reply to this email directly or view it on GitHub.
>